src/julia_internal.h

```diff
@@ -306,6 +306,10 @@ jl_svec_t *jl_perm_symsvec(size_t n, ...);
 // pointer fields.
 #define jl_datatype_layout_n_nonptr(layout) ((uint32_t*)(layout))[-1]
 
+JL_DLLEXPORT void *jl_gc_alloc_array_storage(size_t sz);
+JL_DLLEXPORT void *jl_gc_realloc_array_storage(void *d, size_t sz, size_t oldsz, jl_value_t *owner);
+JL_DLLEXPORT void *jl_gc_counted_realloc_with_align(void *d, size_t sz, size_t oldsz,
```

Comment:
these are hardly the first cases of this so unrelated comment, but I'm a bit confused by things being in julia_internal.h and exported...